From 284375a2457ba6f9c470ee50a39cb2a58243cab4 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Wed, 17 Jun 2020 21:56:07 -0700
Subject: [PATCH 717/830] ANDROID: gki_defconfig: x86: Enable KERNEL_LZ4
X-NVConfidentiality: public

Compress the kernel with lz4 instead of gzip. On x86 this must be
controlled explicitly by kbuild, as decompressor code is prepended
to the compressed kernel artifact.

On arm64 this option is not needed as "make Image.lz4" can be used
instead, or the Image can be compressed in a separate step.

Bug: 150391496
Change-Id: I2d0ce7f937f2a952b4310f53c8b03dc8d721b050
Signed-off-by: Alistair Delva <adelva@google.com>
---
 arch/x86/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 1760750b25bc..4dd18cffd4a0 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -1,5 +1,6 @@
 CONFIG_UAPI_HEADER_TEST=y
 CONFIG_LOCALVERSION="-mainline"
+CONFIG_KERNEL_LZ4=y
 # CONFIG_USELIB is not set
 CONFIG_AUDIT=y
 CONFIG_NO_HZ=y
-- 
2.17.1

